### PR TITLE
func_us_8018CA94

### PR DESCRIPTION
**Pull request by sozud**
Thanks to sonicdcer

This one has an rodata mismatch I think, does it still need to be split to another file even with the rodata migration on?

### CODE REVIEW

**Review by hohle — COMMENTED** [review]
There's a few minor changes that are needed for PSP, but I can post a follow up PR. The major ones are listed below.

File: src/boss/mar/AFC4.c [hunk 1]
```diff
@@ -21,4 +22,185 @@ INCLUDE_ASM("boss/mar/nonmatchings/AFC4", EntityMariaCutscene);
...
+
+    case 13:
+        if ((AnimateEntity(D_us_80180724, self) & 0x80) &&
+            ((self->animFrameIdx == 3) || (self->animFrameIdx == 7))) {
+            posX = self->posX.i.hi - 120;
+            temp = posX >> 4;
+
+            if (posX < 0) {
+                temp = (self->posX.i.hi - 105) >> 4;
+            }
```
[the remark this answers is not in the document]
Comment (sozud) [reply src/boss/mar/AFC4.c:151]: Does your version match in the repo? If you have the rodata part figured out you can just PR your version if it's done

[the remark this answers is not in the document]
Comment (hohle) [reply src/boss/mar/AFC4.c:151]: https://github.com/Xeeynamo/sotn-decomp/pull/1625

**Comment by sozud** [comment]
Closing in favor of https://github.com/Xeeynamo/sotn-decomp/pull/1625

**Pull request closed without merging**